src/deltacode/__init__.py

```diff
@@ -117,11 +118,13 @@ def determine_delta(self):
                     if new_file.sha1 == f.sha1:
                         delta = Delta(0, new_file, f)
                         delta.factors.append('unmodified')
```

Comment:
Now that we have a `status` field for the delta object, we do not need to append that string to our `factors` field.